src/Store/SessionStore.php

```diff
@@ -13,7 +15,7 @@
  *
  * @package SilverStripe\MFA
  */
-class SessionStore implements StoreInterface
+class SessionStore implements StoreInterface, Serializable
```

Comment:
Thoughts on making StoreInterface extend Serializable, so all implementations would be serializable in PHP sessions rather than just our default implementation?

Reply:
I considered this but this implementation is specifically for sessions - I'd imagine that a different `StoreInterface` implementation wouldn't necessarily save in session so probably shouldn't have `Serializable` as a requirement.